config.sh

```diff
@@ -19,6 +19,21 @@ function build_wheel {
     wrap_wheel_builder _build_wheel $@
 }
 
+function pyinst_fname_for_version {
```

Comment:
This is a copy of https://github.com/matthew-brett/multibuild/blob/devel/osx_utils.sh#L147, with a slight tweak. Having a code comment saying this overrides the same thing in `multibuild` and explaining why it's different would be useful.